### PR TITLE
Ks 164 login

### PR DESCRIPTION
**Pull request by preslav-anev**
- Added login functionality.
- Integrated Kount-Access-php-sdk and waiting for Kount's team to get it fully working.
- Added Kount-Access-php-sdk as a composer dependency.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -58,34 +121,22 @@
         <table>
           <h4>Fields Passed</h4>
           <tr>
-            <td><label class="formLabel" for="">PROD_DESC[]</label></td>
-            <td><input class="form-control" type="text" name="desc"
-                       value="Discerning taste, boosting sales  and beating fraud - now that is quite a hat!"
+            <td><label class="formLabel" for="">sessionId</label></td>
+            <td><input class="form-control" type="text" name="session"
+                       value="328457fsd8d7982w3r789erf7"
```
Comment (boyankukushev) [inline index.php]: What's the thing with the session `value`? Why is this hardcoded?

File: index.php [hunk 2]
```diff
@@ -58,34 +121,22 @@
...
-          </tr>
-          <tr>
-            <td><label class="formLabel" for="">PROD_PRICE[]</label></td>
-            <td><input class="form-control" type="text" name="price" value="3999" readonly></input></td>
-          </tr>
-          <tr>
-            <td><label class="formLabel" for="">PROD_QUANT[]</label></td>
-            <td><input class="form-control" type="text" name="item_quantity" value="1" readonly></input></td>
+            <td><label class="formLabel" for="">userHash</label></td>
+            <td><input class="form-control" type="text" name="user" value="fgfoisdfdsf987897fd9s9878s9d7" readonly></input></td>
```
Comment (boyankukushev) [inline index.php]: Why user hash is hardcoded?

File: index.php [hunk 3]
```diff
@@ -58,34 +121,22 @@
...
           </tr>
           <tr>
-            <td><label class="formLabel" for="">PROD_TYPE[]</label></td>
-            <td><input class="form-control" type="text" name="type" value="Hat" readonly></input></td>
+            <td><label class="formLabel" for="">passwordHash</label></td>
+            <td><input class="form-control" type="text" name="password" value="bd0eg809gjv098vf90fcd8g900fd" readonly></input></td>
```
Comment (boyankukushev) [inline index.php]: `password` hardcoded? (and you missed the `hash(username:password`) parameter).

File: success.php [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+$merchantId = $ini["MERCHANT_ID"];
+
+
+const version     = "0210";
+const merchantId  = 999666;
+const host        = merchantId . ".kountaccess.com";
+const accessUrl   = "https://" . host . "/access";
+const apiKey      = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI5OTk2NjYiLCJhdWQiOiJLb3VudC4xIiwiaWF0IjoxNDk5ODcwNDgwLCJzY3AiOnsia2EiOnRydWUsImtjIjp0cnVlLCJhcGkiOnRydWUsInJpcyI6dHJ1ZX19.yFan6moxBonnG8Vk9C_qRpF-eTF00_MRBwgqMdNdy8U";
+const serverUrl   = "api-sandbox01.kountaccess.com";
+const serverUrl2  = "api-sandbox02.kountaccess.com";
```
Comment (boyankukushev) [inline success.php]: probably remove one of the serverUrls ...

**Review by boyankukushev — COMMENTED** [review]
Needs some explanation ...

**Review by leykov — APPROVED** [review]

**Pull request merged**